server/lib/poller.js

```diff
@@ -65,12 +65,12 @@ class Poller
         }
 
         try {
-
             // Check accounts and operations!
-            let checkAccounts = await Config.findOrCreateDefaultBooleanValue(
+            let checkAccounts = false;
+            checkAccounts = await Config.findOrCreateDefaultBooleanValue(
                 'weboob-auto-merge-accounts'
             );
-
+        } finally {
```

Comment:
Keeping the finally, because I want the polling to go on, even if the the parameter 'weboob-auto-merge-accounts'  is not gotten.
Adding an additional try/catch